improvements to getting started tutorial (#5)
sortxyz/docs: docs/tutorials/quick-start.mdx

5. **Create an Issue:**  
   - In the [Data Explorer](/docs/data-explorer/data-explorer), click the Kabob button (the three dots).  
   - Select `Create Issue`, fill in the necessary details, and submit your first [Issue](/docs/category/issues).  

   <img style={{marginLeft:'15px', marginBottom: '5px', height: '140px'}} src={ImgKabob} />

---

## Managing Data

### Proposing Changes

1. **Visit the Data Explorer:**  
   Use the `Explore` button or navigate to 'Data Explorer' via the left-hand menu.  

2. **Editing Existing Data:**  
   - Click the `Edit` button next to the row you want to modify.  
   - Suggest your changes, then create a **Change Request**.  

3. **Adding New Rows:**  
   - Scroll to the bottom of the table and click `Add Row`.  
   - Fill in the new data and submit a **Change Request**.

4. **Removing Data:**  
   - Use the `Edit` button on the relevant row, propose a removal, and submit a **Change Request**.

---

### What Are Change Requests? 🎉

If you're familiar with GitHub Pull Requests, think of **Change Requests** as a similar workflow for making database changes.  

> - **Approval Required:** Changes are not applied to the database until a team member approves them.  

Reviewer: ```suggestion
- **Approval Required:** Changes cannot be applied to the database until a team member approves them.  
```

> - **Data Quality Matters:** By using **Change Requests**, Sort adopts engineering best practices to ensure your data remains accurate and reliable.

Reviewer: ```suggestion
- **Data Quality Matters:** By using **Change Requests**, Sort adopts software engineering best practices to ensure your data remains accurate and reliable.
```

---

## Fun Stuff! 🎉

- **Manage Workflows:** Relate Issues to Change Requests for streamlined operations.  

> - **Collaborate with Teams:** Use Sort's tools for Customer Support, Data Teams, Internal Operations, or Community Managers.

Reviewer: The Fun Stuff section should be deleted b/c it is not actionable.

Author: +1 Removed
